workflow-templates/devonfw-sync-docs.yml

```diff
@@ -0,0 +1,71 @@
+name: Devonfw CI for doc sync
+env:
+  REPO_CONSOLIDATE: "devonfw-guide"
+  BUILD_USER: ${{ secrets.BUILD_USER }}
+  BUILD_USER_PASSWD: ${{ secrets.BUILD_USER_PASSWD }}
+  BUILD_USER_EMAIL:  ${{ secrets.BUILD_USER_EMAIL }}
+on:
+  push:
+    branches:
+      - $default-branch
+name: Sync Docs
+jobs:
+  update-wiki:
+    if: ${{github.repository == 'devonfw/ide'}}
```

Comment:
make sure here is written the target repository. There might be a avariable to be put here instead of hard coding devonfw/ide

Reply:
fixed this.